### ISSUE TITLE
The story tab is blank (MacOS 10.12 Sierra)

### ISSUE THREAD

**Issue by thSoft** [TobyLobster/Inform#2]
http://inform7.com/mantis/view.php?id=1953 was fixed, however there has been no release since then (and I couldn't comment on the issue on the bug tracker as a reporter). Please build a version for macOS 10.12.

**Comment by greatquux** [comment]
And I would be very happy if that same person could also build a new version of Zoom that fixes this bug. I've spent about 4+ hours fooling around with Xcode, and learning about all the great (!) changes that have been made from the last time this was compiled, and it hasn't been fun.  Just when I do work around one bug, I hit another. Even after compilation I run into problem starting the Sparkle framework and at that point I've decided to give up. ;)
